### PR TITLE
Mcs 2930 command serve format

### PR DESCRIPTION
**Pull request by anchi1216**
@dariachen @kevingo 

新增command server documentation in key concept chapter.

### CODE REVIEW

File: content/en/key_concept/README.md [hunk 1]
```diff
@@ -144,3 +144,53 @@ Once you beta-release your prototype, the prototype will be frozen, which means
...
+
+## Maintaining heartbeat
+
+Once get the server ip and port to connect, the device need to send a heartbeat to the command server to be identified. The device also need to sent heartbeats to the server every 120 seconds to stay connected, or the server will disconnect the device.
+
+Heartbeat format:
+
+```
+    deviceId,deviceKey,timestamp
+
```
Comment (appleboy) [inline content/en/key_concept/README.md:178]: 前面不用空白 直接寫 

> ```
> deviceId,deviceKey,timestamp
> ```

File: content/zh-TW/key_concept/README.md [hunk 2]
```diff
@@ -161,3 +161,53 @@ MCS 提供了以下多種資料通道，以供使用者操作遠端設備或顯
...
+
+將您的裝置與 command server 做連接，您即可以透過 MCS 頁面來控制您的裝置。若您不僅僅希望只將裝置接收的資料點上傳
+至 MCS，並且希望同時能透過 MCS 發出指令控制您的裝置，您必須將裝置與 MCS 的 command server 做連接。
+
+以下是將裝置與 command server 做連接的步驟：
+
+1. 建立裝置與 command server 之間的 TCP 長連結。
+3. 維持 heart beat 訊號。
+4. 透過 MCS 頁面對裝置下指令。
+
```
Comment (appleboy) [inline content/zh-TW/key_concept/README.md:174]: 1, 3, 4 ?

File: content/zh-CN/key_concept/README.md [hunk 3]
```diff
@@ -152,3 +152,52 @@ MCS 提供了以下多种资料通道，以供使用者操作远程设备或显
...
+
+将您的装置与 command server 做连接，您即可以透过 MCS 页面来控制您的装置。若您不仅仅希望只将装置接收的资料点上传
+至 MCS，并且希望同时能透过 MCS 发出指令控制您的装置，您必须将装置与 MCS 的 command server 做连接。
+
+以下是将装置与 command server 做连接的步骤：
+
+1. 建立装置与 command server 之间的 TCP 长连结。
+3. 维持 heartbeat 讯号。
+4. 透过 MCS 页面对装置下指令。
+
```
Comment (appleboy) [inline content/zh-CN/key_concept/README.md:165]: 1 3 4

File: content/en/key_concept/README.md [hunk 4]
```diff
@@ -144,3 +144,53 @@ Once you beta-release your prototype, the prototype will be frozen, which means
...
+
+# Command Server
+
+Command server is for you to give command to the device from MCS. If you wish not only to upload data to MCS but also need to control your device using the MCS console, you have to connect the device to the command server in order to give command to device through MCS.
+
+Here is the steps to connect your device to MCS:
+1. Build a TCP long connection tetween command server and device.
+3. Maintaining heart beat.
+4. Give command through MCS console.
```
Comment (appleboy) [inline content/en/key_concept/README.md]: 1 3 4

**Comment by anchi1216** [comment]
@kevingo @appleboy 感謝 皆已更正

**Comment by anchi1216** [comment]
更正~
1. 格式
2. 拿掉 "覆"
3. connection

**Comment by kevingo** [comment]
LGTM

**Pull request merged**